docs/tutorials/model-registry.txt

```diff
@@ -0,0 +1,245 @@
+.. _organizing-models:
+
+Organizing Models in the Model Registry
+=======================================
+
+In this tutorial, we focus on organization of checkpoints via the Determined
+Model Registry. Before reading this guide, we recommend that users first read
+the :ref:`use-trained-models` how to guide which describes how to interact
+with checkpoints.
+
+
+Managing Models
+---------------
+
+A model is a named set of checkpoints. Each model in Determined must have a
```

Comment:
"A model is a named set of checkpoints." -> I'd quibble with this slightly: if a model *is* a named set of checkpoints, it logically shouldn't include any additional metadata.

Should we also clarify the use of the term "model" in this doc and how that differs from, say, the "model definition" we talk about elsewhere in the docs/system?

Reply:
We added the metadata at the model level to capture anything that is relevant for all versions of the model as shown in the example. Is there different language you would prefer to describe the relationship between a model and its versions?